tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -24,73 +29,117 @@ ResourceInfo:
 
 Observers:
   VolumeFileName: "CharacteristicExtractUnusedVolume"
+  # The reduction file is where the CCE output will be written.
+  # Specifically, it will be in a `/SpectreR0257.cce` where the number is the
+  # ExtractionRadius specified below.
   ReductionFileName: "CharacteristicExtractReduction"
 
 EventsAndTriggers:
+  # Write the CCE time step every Slab. A Slab is a collection of time steps,
+  # where the time step size can be adjusted dynamically.
   - Trigger:
       Slabs:
         EvenlySpaced:
           Offset: 0
           Interval: 1
     Events:
       - ObserveTimeStep:
+          # The output is written into the "ReductionFileName" HDF5 file under
+          # "/SubfileName.dat"
           SubfileName: CceTimeStep
           PrintTimeToTerminal: true
 
 Cce:
   Evolution:
     TimeStepper:
       AdamsBashforth:
-        Order: 3
+        Order: 3 # Going to higher order doesn't seem necessary for CCE
     StepChoosers:
-      - Constant: 0.5
+      - Constant: 0.5 # Don't take steps bigger than 0.5M
```

Comment:
We might want to lower this to 0.1 for production runs.

Reply:
Okay, I'll decrease it! We hopefully can have the tests not time out with that so then we can just have the sample not require changes for production :)